cmake/fuzzing_options.cmake

```diff
@@ -16,11 +16,18 @@
 
 include(compiler_id)
 
-option(FUZZING "Build for Fuzz Testing (local fuzzing and OSS Fuzz)" OFF)
+option(FUZZING "Build for local Fuzz Testing" OFF)
+option(OSS_FUZZ "Build for OSS Fuzz Environment" OFF)
```

Comment:
Is this necessary? Could we detect that OSS Fuzz is in effect by the presence of their environment variable? (I'd suggest we still require FUZZING to be ON for either case.)

Reply:
No not necessary, removed and detected from `ENV{LIB_FUZZING_ENGINE}` when needed.